experiments/azure/cyclecloud/cpu/README.md

```diff
@@ -0,0 +1,211 @@
+# AWS Parallel Cluster
+
+This setup study aims to prototype using [AWS parallel cluster](https://docs.aws.amazon.com/parallelcluster/latest/ug/install-v3-virtual-environment.html). Note that since this requires nodejs (and I refuse to install this on my system) we will be using a Docker container. It is extended from [our original test](https://github.com/converged-computing/metrics-operator-experiments/tree/main/performance/testing/slurm/aws-0). Build the container:
```

Comment:
This looks like it was copy pasted from parallel cluster and needs to be updated or removed.

Reply:
It's removed in the commit that updates the README.